<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting7Widget</class>
 <widget class="QWidget" name="Setting7Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="CustomComboBox" name="comboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="CustomLabel" name="WifiLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="CustomLabel" name="nineAxisSensorLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="CustomLabel" name="micnSpeackerLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="CustomLabel" name="blueToothLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="CustomLabel" name="powerSavingTimeLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="CustomComboBox" name="comboBox_2"/>
     </item>
     <item row="2" column="1">
      <widget class="CustomComboBox" name="comboBox_3"/>
     </item>
     <item row="3" column="1">
      <widget class="CustomComboBox" name="comboBox_4"/>
     </item>
     <item row="4" column="1">
      <widget class="CustomComboBox" name="comboBox_5"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>CustomLabel.h</header>
  </customwidget>
  <customwidget>
   <class>CustomComboBox</class>
   <extends>QComboBox</extends>
   <header>CustomComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
